Gruntfile.js

```diff
@@ -137,7 +141,7 @@ module.exports = function(grunt) {
 
     grunt.registerTask('addPolyfill', function() {
         var fs = require('fs');
-        var polyfillPath = './node_modules/grunt-babel/node_modules/babel-core/browser-polyfill.js';
+        var polyfillPath = './node_modules/babel-polyfill/dist/polyfill.js';
```

Comment:
Agreed, it should be left to the application using this library to determine if they need the polyfill or not.